components/radial-chart/src/radial-chart-path.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, TemplateRef, AfterContentInit } from '@angular/core';
+import { DtOverlayTrigger } from '@dynatrace/barista-components/overlay';
+import { DtRadialChartRenderData } from './utils/radial-chart-interfaces';
+
+export interface DtRadialChartOverlayData {
+  name: string;
+  value: number;
+  totalValue: number;
+}
+
+@Component({
+  selector: 'svg:g[dt-radial-chart-path]',
+  templateUrl: 'radial-chart-path.html',
+  host: {
+    '(mouseenter)': 'overlayTemplate && _handleMouseEnter($event)',
+    '(mouseleave)': 'overlayTemplate && _handleMouseLeave($event)',
+  },
+})
+export class DtRadialChartPath
+  extends DtOverlayTrigger<{ $implicit: DtRadialChartOverlayData }>
+  implements AfterContentInit {
+  @Input() series: DtRadialChartRenderData;
```

Comment:
Please add jsdoc comments to all public properties, even if this is an internal component and is not being exported to the consumer.